android/build.gradle

```diff
@@ -1,6 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     apply from: rootProject.file("gradle/config.gradle")
+    ext.kotlin_version = '1.3.31'
```

Comment:
Consider naming it "kotlinVersion" to keep the same style with other variables. And if this is a global variable, consider moving it to `config.gradle`.

Reply:
Thanks, have removed this name.